client/blocks/inline-help/inline-help-rich-result.jsx

```diff
@@ -0,0 +1,132 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+import { omitBy, isUndefined } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import { RESULT_ARTICLE, RESULT_TOUR, RESULT_VIDEO } from './constants';
+import Button from 'components/button';
+import Dialog from 'components/dialog';
+import ResizableIframe from 'components/resizable-iframe';
+import { decodeEntities, preventWidows } from 'lib/formatting';
+import { recordTracksEvent } from 'state/analytics/actions';
+import { getSearchQuery } from 'state/inline-help/selectors';
+import { requestGuidedTour } from 'state/ui/guided-tours/actions';
+
+class InlineHelpRichResult extends Component {
+	static propTypes = {
+		result: PropTypes.object,
+	};
+
+	state = {
+		showDialog: false,
+	};
+
+	handleClick = event => {
+		event.preventDefault();
+		const { href } = event.target;
+		const { type } = this.props;
+		const { tour } = this.props.result;
+		const tracksData = omitBy(
+			{
+				search_query: this.props.searchQuery,
+				tour,
+				result_url: href,
+			},
+			isUndefined
+		);
+
+		this.props.recordTracksEvent( `calypso_inlinehelp_${ type }_open`, tracksData );
+
+		if ( type === RESULT_TOUR ) {
+			this.props.requestGuidedTour( tour );
+		} else if ( type === RESULT_VIDEO ) {
+			if ( event.metaKey ) {
+				window.open( href, '_blank' );
```

Comment:
I'd expect this to open the "regular" YouTube link (instead of the embed). Not a blocker, but it'd be nice to have :)

Reply:
Yes, I think we'll change the link in the data source file to just contain the video ID. Then we can either use the embed url or the standard url. It'll also allow us to use the thumbnails individually if we want to.

Reply:
Let's do this in a seperate PR as it affects the data file that has been edited independently of this PR. Noted it down as an iteration TODO.